pkg/platform/runtime/store/store.go

```diff
@@ -187,7 +187,8 @@ func (s *Store) CommitID() (string, error) {
 
 	lines := strings.Split(string(contents), "\n")
 	if len(lines) < 1 {
-		return "", locale.NewError("err_deploy_uninstall_marker", "Marker file is incomplete, cannot determine commitID")
+		// Older installations may not have a marker file
```

Comment:
So then wouldn't the error on line 185 have triggered? I think what you're addressing here is that the file exists but doesn't have the data you're looking for.